docs/language-server/protocol-language-server.md

```diff
@@ -2302,8 +2323,20 @@ an execution context.
 
 ```typescript
 {
+  /**
+   * The identifier of the execution context.
+   */
   contextId: ContextId;
+
+  /**
+   * The error message.
+   */
   message: String;
```

Comment:
Errors should be able to be richer than just a `String`. We want to be able to output nicely formatted JSON errors at some point, and the protocol should be flexible enough to cope with this.

Reply:
Language Server protocol is already encoded as JSON. You probably meant the command line execution, which can be extended to output stuff to console using different formats, but that's a different task.